src/adapters/mc/asb/AzureLinuxBaseline_DeployIfNotExists.json.tmpl

```diff
@@ -14,8 +14,8 @@
         "name": "AzureLinuxBaseline",
         "version": "1.0.0",
         "contentType": "Custom",
-        "contentUri": "https://github.com/Azure/azure-osconfig/releases/download/test_policy_package/AzureLinuxBaseline.zip",
-        "contentHash": "832539A6E9167E356B1CA99F090E20B6C94673FAB6E6714881798C56D8EBD44E"
+        "contentUri": "@URI@",
+        "contentHash": "@HASH@"
       }
```

Comment:
Please do not change these test policy definitions. They need to target specific packages. They are here only for reference.